sonic-net/sonic-buildimage
# [SAI-PTF][202211]enable sai-ptf logger in sai_adapter to log all the sai api invcations

Signed-off-by: richardyu-ms <richard.yu@microsoft.com>



#### Why I did it
enable sai-ptf logger in sai_adapter to log all the sai api invcations
#### How I did it
add build parameter to enable the sai-ptf logger when build sai PRC
#### How to verify it
How to verify it
local build test
test the generated sai_adapter
test with pipeline
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
